webui/react/src/pages/TaskList.tsx

```diff
@@ -155,11 +155,11 @@ const TaskList: React.FC = () => {
         plural: '',
         sources: [] as number[],
       };
-      if (record.misc?.experimentIds) {
+      if (record.misc?.experimentIds && record.misc?.experimentIds.length !== 0) {
```

Comment:
is this from another branch?

Reply:
oh yeah, I need to get rid of this. thanks!